docs/Guides/ThePackageManager.md

```diff
@@ -2,56 +2,83 @@
 
 ## Installing the package manager
 
-Download the latest [meta pax](https://github.com/ZOSOpenTools/meta/releases) to z/OS
+If you have curl and bash on your system, you can use this one liner:
```

Comment:
, and connectivity to github.com, githubusercontent.com, and mayb other related domains :)

Reply:
That is true for using zopen in general, probably need a pre-req section in there